src/main/java/seedu/duke/member/TeamMember.java

```diff
@@ -1,7 +1,7 @@
 package seedu.duke.member;
 
 public class TeamMember {
-    protected String name;
+    public String name;
```

Comment:
I don't think this should be private too

Reply:
Sean why ah ? We making all the attributes in the other classes private anyway right